packages/cozy-mespapiers-lib/src/components/Multiselect/ForwardModal.jsx

```diff
@@ -94,7 +93,7 @@ const ForwardModal = ({ onClose, onForward, file }) => {
         ttl,
         password
       })
-      copyToClipboard(url, { target: modalContentRef.current, t, showAlert })
+      copyToClipboard(url, { t, showAlert })
```

Comment:
la méthode est maintenant async, ne devrait-on pas faire un await dessus ? 🤔 (même remarque partout)

Reply:
J'ai changé, c'est plus cohérent d'await même si parfois il n'y avait rien derrière

Reply:
En passant, vu qu'on avait utilisé dans mes papiers, la lib qu'on utilisait dans cozy-sharings pour copier le lien (si je ne me trompe pas), peut-on aussi changer dans cozy-sharings ?

Reply:
Oui j'ai remarqué aussi je me le suis noté

Reply:
https://github.com/cozy/cozy-libs/pull/2425